### PR TITLE
Edit Run Modal

### PR DESCRIPTION
**Pull request by sam-kakengi**

### CODE REVIEW

**Review by Dimwiddle — APPROVED** [review]

**Comment by Dimwiddle** [comment]
Looks good! Just have a look at the merge conflict before merging @sam-kakengi

**Comment by sam-kakengi** [comment]
Sorry, also changed the running man icon earlier
Screenshot 2024-10-07 143811
 also

File: frontend/src/components/core-components/dashboard/editRunComponents/EditRunModal.js [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+        }
+        onRunUpdated(updatedRun)
+        handleClose()
+      } else {
+        console.error('Failed to update run')
+      }
+    } catch (error) {
+      console.error('Error updating run')
+    }
+  }
```
Comment (Dimwiddle) [inline frontend/src/components/core-components/dashboard/editRunComponents/EditRunModal.js:132]: Can remove the logging. Don't wanna show this in production

**Comment by Dimwiddle** [comment]
> Sorry, also changed the running man icon earlier also
> 
> Screenshot 2024-10-07 143811
> 
> 

Looks good!

Did you test at different dimensions? @sam-kakengi

**Comment by sam-kakengi** [comment]
> > Sorry, also changed the running man icon earlier also
> > Screenshot 2024-10-07 143811
> 
> Looks good!
> 
> Did you test at different dimensions? @sam-kakengi

Yeah it was just a bit too big at medium screen size but fixed that

**Comment by Dimwiddle** [comment]
You can go ahead and squash merge this PR.

We'll do a review and UAT before deploying.

Nice work @sam-kakengi 🙌👏

**Comment by sam-kakengi** [comment]
Appreciate the feedback a lot, thanks! Look forward to it!

**Pull request merged**